### DISCUSSION TITLE
allow Awards DXCC to search across more than one profile (ones for same user same country)

### DISCUSSION THREAD

**Discussion by AD7Djoe** [magicbug/Cloudlog#459] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I am the same operator working in the same country (such as for ARRL DXCC Rule #10) having two separate profiles (one temporary for OH/AD7D and the other when I got my official call OG7D) and I have to go switch to each profile to find countries I have worked for DXCC.

To make matters worse, each profile separates the counts.

**Describe the solution you'd like**
Awards DXCC should take in account that per ARRL DXCC rule #10, I am the same operator in the same country and combine my DXCC listing results.

**Describe alternatives you've considered**
The only alternative now is when working as OG7D is to switch my active profile back to OH/AD7D look at Awards DXCC, see if I have worked a country, then switch back to OG7D profile and then work the station.

This is very cumbersome.

Other alternative is to maintain my own DXCC list in another document.

**Additional context**

**Comment by ok1cdj** [comment]
That would  be great to have this feature.
